_notices/rgn0001.md

```diff
@@ -36,9 +36,10 @@ For more information read the [GitHub docs](https://github.com/github/renaming/)
 
 ### Updates
 
-- 10-Jul-2020 - `main` branches were added to all repos with an existing stable/release branch
-- 15-Jul-2020 - Documentation with outdated references were updated with PRs to `main`
-- On-going - PRs to update scripts and other mentions will be merged into `branch-0.15` and then into `main` upon `v0.15` release
+- **10-Jul-2020** - `main` branches were added to all repos with an existing stable/release branch
+- **15-Jul-2020** - Documentation with outdated references were updated with PRs to `main`
+- **Completed** - PRs to update scripts and other mentions will be merged into `branch-0.15` and then into `main`
+- **On-going** - Release `v0.15` is occurring and using the `main` branch; the old branches will be removed post-release
```

Comment:
trivial, but I'm accustomed to seeing `ongoing` of on-going.

https://www.merriam-webster.com/dictionary/ongoing